mmpose/datasets/pipelines/top_down_transform.py

```diff
@@ -298,8 +298,8 @@ def _generate_target(self, cfg, joints_3d, joints_3d_visible):
                     target[joint_id][img_y[0]:img_y[1], img_x[0]:img_x[1]] = \
                         g[g_y[0]:g_y[1], g_x[0]:g_x[1]]
 
-        if use_different_joint_weights:
-            target_weight = np.multiply(target_weight, joint_weights)
+        if use_different_joints_weight:
```

Comment:
emmmm

Reply:
it's top_down_transform.py ,i modified top_down_crowd_transform.py. Should i modify these files now ? or you process when rebase master

Reply:
could you try to rebase master?

Reply:
okay, if you have problems in rebase, I will do it later